exchange/exchange_test.go

```diff
@@ -598,6 +622,66 @@ func TestCategoryMapping(t *testing.T) {
 	assert.Equal(t, 3, len(bidCategory), "Bidders category mapping doesn't match")
 }
 
+func TestCategoryMappingNoIncludeBrandCategory(t *testing.T) {
+
+	categoriesFetcher, error := newCategoryFetcher("./test/category-mapping")
+	if error != nil {
+		t.Errorf("Failed to create a category Fetcher: %v", error)
+	}
+
+	requestExt := newExtRequestNoBrandCat()
+
+	targData := &targetData{
+		priceGranularity: requestExt.Prebid.Targeting.PriceGranularity,
+		includeWinners:   true,
+	}
+	durationRange := make([]int, 0)
+	durationRange = append(durationRange, 15)
+	durationRange = append(durationRange, 30)
+	durationRange = append(durationRange, 50)
+	requestExt.Prebid.Targeting.DurationRangeSec = durationRange
```

Comment:
This can be a one-liner as: `requestExt.Prebid.Targeting.DurationRangeSec = []int{15,30,50}`